.github/workflows/build.yaml

```diff
@@ -1,15 +1,16 @@
 name: build
 
 on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
```

Comment:
Let's remove the paths-ignore keyword for now as it doesn't play well with branch protection rules.